src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConditionalOperation.cs

```diff
@@ -161,5 +155,235 @@ void M(bool a, bool b)
+                  ILiteralOperation (OperationKind.Literal, Type: System.Int32, Constant: 0) (Syntax: '0')
+
+    Jump if False to Block[5]
+        IParameterReferenceOperation: a (OperationKind.ParameterReference, Type: System.Boolean) (Syntax: 'a')
+
+    Next Block[2]
+Block[2] - Block
+    Predecessors (1)
+        [1]
+    Statements (0)
+    Jump if False to Block[4]
+        IParameterReferenceOperation: b (OperationKind.ParameterReference, Type: System.Boolean) (Syntax: 'b')
+
+    Next Block[3]
+Block[3] - Block
+    Predecessors (1)
+        [2]
+    Statements (1)
+        IFlowCaptureOperation: 1 (OperationKind.FlowCapture, Type: null, IsImplicit) (Syntax: 'c')
+          Value: 
+            IParameterReferenceOperation: c (OperationKind.ParameterReference, Type: System.Boolean) (Syntax: 'c')
+
+    Next Block[8]
+Block[4] - Block
+    Predecessors (1)
+        [2]
+    Statements (1)
+        IFlowCaptureOperation: 1 (OperationKind.FlowCapture, Type: null, IsImplicit) (Syntax: 'b')
+          Value: 
+            ILiteralOperation (OperationKind.Literal, Type: System.Boolean, Constant: False, IsImplicit) (Syntax: 'b')
+
+    Next Block[8]
+Block[5] - Block
+    Predecessors (1)
+        [1]
+    Statements (0)
+    Jump if True to Block[7]
+        IParameterReferenceOperation: d (OperationKind.ParameterReference, Type: System.Boolean) (Syntax: 'd')
+
+    Next Block[6]
+Block[6] - Block
+    Predecessors (1)
+        [5]
+    Statements (1)
+        IFlowCaptureOperation: 1 (OperationKind.FlowCapture, Type: null, IsImplicit) (Syntax: 'e')
+          Value: 
+            IParameterReferenceOperation: e (OperationKind.ParameterReference, Type: System.Boolean) (Syntax: 'e')
+
+    Next Block[8]
+Block[7] - Block
+    Predecessors (1)
+        [5]
+    Statements (1)
+        IFlowCaptureOperation: 1 (OperationKind.FlowCapture, Type: null, IsImplicit) (Syntax: 'd')
+          Value: 
+            ILiteralOperation (OperationKind.Literal, Type: System.Boolean, Constant: True, IsImplicit) (Syntax: 'd')
+
+    Next Block[8]
+Block[8] - Block
+    Predecessors (4)
+        [3]
+        [4]
+        [6]
+        [7]
+    Statements (1)
+        IExpressionStatementOperation (OperationKind.ExpressionStatement, Type: null) (Syntax: 'GetArray()[ ... : (d || e);')
+          Expression: 
+            ISimpleAssignmentOperation (OperationKind.SimpleAssignment, Type: System.Boolean) (Syntax: 'GetArray()[ ...  : (d || e)')
+              Left: 
+                IFlowCaptureReferenceOperation: 0 (OperationKind.FlowCaptureReference, Type: System.Boolean, IsImplicit) (Syntax: 'GetArray()[0]')
+              Right: 
+                IFlowCaptureReferenceOperation: 1 (OperationKind.FlowCaptureReference, Type: System.Boolean, IsImplicit) (Syntax: 'a ? (b && c) : (d || e)')
+
+    Next Block[9]
+Block[9] - Exit
+    Predecessors (1)
+        [8]
+    Statements (0)
+";
+            var expectedDiagnostics = DiagnosticDescription.None;
```

Comment:
Nit: consider making `expectedDiagnostics` an optional value in `VerifyFlowGraphAndDiagnosticsForTest` since most tests expect no diagnostics.

Reply:
Given that these are test helpers, it is much better when the value is passed explicitly, people actually have to think what it should be and reviewers don't have to guess what optional parameters the helper has.

---
In reply to: [163072954](https://github.com/dotnet/roslyn/pull/24358#discussion_r163072954) [](ancestors = 163072954)